src/sentry/tagstore/types.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+from sentry.tagstore.base import TagKeyStatus
+
+
+class TagKey(object):
+    def __init__(self, key, values_seen, status=TagKeyStatus.VISIBLE):
+        self.key = key
+        self.values_seen = values_seen
+        self.status = status
+
+
+class TagValue(object):
+    def __init__(self, key, value, times_seen, first_seen, last_seen):
+        self.key = key
+        self.value = value
+        self.times_seen = times_seen
+        self.first_seen = first_seen
+        self.last_seen = last_seen
+
+
+class GroupTagKey(object):
+    def __init__(self, group_id, key, values_seen):
+        self.group_id = group_id
+        self.key = key
+        self.values_seen = values_seen
+
+
+class GroupTagValue(object):
+    def __init__(self, group_id, key, value, times_seen, first_seen, last_seen):
+        self.group_id = group_id
+        self.key = key
+        self.value = value
+        self.times_seen = times_seen
+        self.first_seen = first_seen
+        self.last_seen = last_seen
+
+
+from sentry.api.serializers import Serializer, register
+
+
+@register(TagKey)
+class TagKeySerializer(Serializer):
+    def serialize(self, obj, attrs, user):
```

Comment:
Do these even need to be in their own class? Can `TagKey` itself be `@register`ed as its own serializer as long as it has a `serialize` method?